usaspending_api/download/management/commands/resolve_frozen_downloads.py

```diff
@@ -0,0 +1,102 @@
+import dateutil
+import logging
+import pytz
+import time
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+
+from usaspending_api.download.lookups import JOB_STATUS_DICT
+from usaspending_api.download.models import DownloadJob
+from usaspending_api.download.v2.download_admin import DownloadAdministrator
+
+logger = logging.getLogger("console")
+
+
+class Command(BaseCommand):
+    help = " ".join(
+        [
+            "Restart all 'frozen' download jobs which meet the provided filters",
+            "  (frozen signifies it didn't complete or fail, and is not a monthly download job)",
+            "Ability to filter by an open or closed date range, job status, and if it is a monthly record",
+            "Depending on environment settings, this will either re-queue the download or process locally",
+        ]
+    )
+
+    def add_arguments(self, parser):  # used by parent class
+        parser.add_argument(
+            "-s",
+            "--start",
+            "--start-date",
+            "--start-datetime",
+            dest="start",
+            type=dateutil.parser.parse,
+            help="Earliest datetime to begin restarting downloads",
+        )
+        parser.add_argument(
+            "-e",
+            "--end",
+            "--end-date",
+            "--end-datetime",
+            dest="end",
+            type=dateutil.parser.parse,
+            help="Most recent datetime to restart downloads",
```

Comment:
Might be useful to say that the current date (now) will be used if not set (i.e. no upper limit)

Reply:
That's technically not correct. If end isn't provided, the date range is "open". There are some notes in the command help documentation.